lms/djangoapps/teams/views.py

```diff
@@ -541,6 +562,15 @@ def get_queryset(self):
         """Returns the queryset used to access the given team."""
         return CourseTeam.objects.all()
 
+    def delete(self, request, team_id):
+        """DELETE /api/team/v0/teams/{team_id}"""
+        team = get_object_or_404(CourseTeam, team_id=team_id)
+        self.check_object_permissions(request, team)
+        # Note: also deletes all team memberships associated with this team
+        log.info('user %d deleted team %s', request.user.id, team_id)
```

Comment:
Nit: put the log after the call to ``team.delete`` so that we don't log if the delete fails.